---
layout: archive
title: "Research in the lab"
author_profile: false
permalink: /
---

### **Adaptation to abiotic stress **
My research investigates how crops adapt to abiotic stresses such as salinity, drought and heat, all of which impose strong constraints on growth and yield. These stresses act as powerful selective pressures, shaping the genetic and regulatory architecture of plant populations. Biologically, abiotic stresses drive selection by favoring alleles that enhance survival and performance but often at a cost to other fitness traits. For example, salinity stress can intensify selection on gene expression networks, while domestication has historically favored loss of resistance or stress-response genes to optimize yield. Such trade-offs highlight the evolutionary constraints and adaptive potential of crops facing environmental pressures. I am interested in identifying key phenotypes, genes and regulatory networks that is associated with response and adaptation (including trade-offs) to abiotic stresses. I use a suite of approaches to quantify how abiotic stresses act as selective agents, and shapes genetic variation and regulatory networks that underpin crop adaptation. By studying adaptation at the intersection of physiology, genomics, and evolution, I aim to uncover mechanisms that can be harnessed for crop improvement under changing climates.
<br/>
### **Divergence during domestication and post-domestication divergence**
Another area that I am interested in is understanding how domestication and post-domestication processes have shaped genome evolution and driven divergence both between wild progenitors and cultivated crops, as well as among different cultivated lineages. My research focuses on uncovering the how the interplay of processes such as selection, demographic bottlenecks, gene loss, structural variation, and introgression, shape the diversity we see today. By integrating comparative genomics, population genetics, and functional analyses, I identify genetic variants, structural changes, and regulatory shifts that underlie domestication and post-domestication diversification. This work not only deepens our understanding of plant evolution but also gives us insights into the history of domestication and its spread. Further, it provides critical genomic resources for linking evolutionary processes to key agronomic traits, thereby accelerating efforts in crop adaptation and improvement.
<br/>
<br/>
Methodologically, I employ large-scale field and greenhouse phenotyping combined with multi-omics (genomics, transcriptomics and epigenomics). This includes comparative genomics, population genetics, association mapping, and selection scans to identify genes and regulatory regions impacted by the various evolutionary processes such as selection (natural and artificial), bottlenecks, drift, and introgression. 

## **Study Systems**
# **Pigeonpea**   
Pigeonpea *(Cajanus cajan)*, commonly known as Arhar or Tur dal, is an important diploid outcrossing legume (2n = 22) which was domesticated relatively recently (3500 years ago) in India. It is a nutrient dense crop that is cultivated on over 6.36 million hectare land area globally (FAO Stat. 2022, particularly in the semi-arid regions of India and East Africa. Although more resilient to abiotic stress than most crops, pigeonpea productivity suffers due to high salinity and temperature extremes. Despite its economic importance and resilience in marginal environments, pigeonpea's genetic basis for tolerance and adaptation to environmental stressors remains poorly understood -- an area my lab is actively investigating.
-----
<div style="float: center; padding-right: 1px">
    <a href="http://gupta-plantgenevo.github.io/images/pigeonpea.png"><img src="/images/pigeonpea.png" title="Pigeonpea shrub (P.C: ICRISAT)" width="300" border="0" onClick="_gaq.push(['_trackEvent', 'IMGs', 'Image', 'grandiflora']);"></a>
</div>

# **Rice**
Rice *(Oryza sativa L.)* is a staple cereal crop and one of the most important food sources worldwide, feeding more than half of the global population. It is a diploid selfing species (2n = 24) that was domesticated over 9,000 years ago from its wild progenitor Oryza rufipogon in Asia. Rice serves as a model system for monocot genomics, offering extensive genomic resources to study domestication, adaptation, and responses to environmental stress. Rice comprises diverse varieties, most notably the indica and japonica, and the hybrid varieties -- basmati and aus, which differ in physiology, grain quality, and environmental adaptability. The lab is interested in studying the hybridization history in rice and understanding the contribution of the parental varieties in the formation of the hybrid varieties. 
In addition, rice is grown using different cultivation systems (upland-rainfed, lowland-rainfed, irrigated and deep-water) which have led to distinct selection pressures that shaped genomic and regulatory diversity. The lab is interested in uncovering this diversity.
-----
<div style="float: center; padding-right: 1px">
    <a href="http://gupta-plantgenevo.github.io/images/rice.jpg"><img src="/images/rice.jpg" title="Rice field experiment, IRRI (P.C: Simon C. Groen)" width="400" border="0" onClick="_gaq.push(['_trackEvent', 'IMGs', 'Image', 'grandiflora']);"></a>
</div>
